config.yaml

```diff
@@ -0,0 +1,14 @@
+  env: development
```

Comment:
please add .gitignore to ignore `config.yaml`. Instead, create `config.yaml.example` to define all config variables